src/modules/src/Kconfig

```diff
@@ -62,6 +62,21 @@ endmenu
 
 menu "Motor configuration"
 
+choice
+    prompt "ESC protocol"
+    default MOTORS_ESC_PROTOCOL_STANDARD_PWM
+
+	config MOTORS_ESC_PROTOCOL_STANDARD_PWM
+	    bool "Standard PWM"
```

Comment:
I named the default protocol "standard PWM" based on [this article](https://quadmeup.com/pwm-oneshot125-oneshot42-and-multishot-comparison/). Feel free to rename if this is not the name that you would prefer.